Makefile

```diff
@@ -360,7 +360,8 @@ manifests/base/crds/full/argoproj.io_workflows.yaml: $(GOPATH)/bin/controller-ge
 	kustomize version
 
 # generates several installation files
-manifests/install.yaml: $(CRDS) /usr/local/bin/kustomize
+.PHONY: manifests
+manifests: $(CRDS) /usr/local/bin/kustomize
```

Comment:
It seems that if we use `manifests/install.yaml` for this target it will not be refreshed if the file already exists (which is always the case)